sdk/src/main/res/layout/consent_mode_selection_fragment.xml

```diff
@@ -0,0 +1,147 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<android.support.constraint.ConstraintLayout
```

Comment:
Could this layout potentially use single constraint layout as parent?

Reply:
Will have a play around and see.

Reply:
It seems that the card view relies on this to intrinsically calculate it's size, as well as the position of the guidelines and subsequent layout attributes, so in this case, I believe we need the nested layout. That said, you're better than me with layouts so if you think you can make it work then feel free to have a crack at it. :)